build_tools/github_actions/lint_shellcheck.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright 2023 The StableHLO Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This runs buildifier on relevant files
```

Comment:
Comment outdated?

Reply:
Ah, does this run all shell linting scripts?

Could you add a description for that shellcheck is / means?

Reply:
Thanks for pointing that out! This will lint all shell scripts with shellcheck. I added a description of what shellcheck is.